docs/sidebars.ts

```diff
@@ -149,10 +149,10 @@ module.exports = {
     },
     {
       "type": "category",
-      "label": "Integrations",
+      "label": "Integrating Dagger with CI",
```

Comment:
`Integrating with CI` reads better to me.

Reply:
Done